src/System.Private.ServiceModel/src/System/ServiceModel/Channels/RTSocketConnection.cs

```diff
@@ -190,7 +190,7 @@ protected override void ShutdownCore(TimeSpan timeout)
             catch (ObjectDisposedException objectDisposedException)
             {
                 Exception exceptionToThrow = ConvertObjectDisposedException(objectDisposedException, TransferOperation.Undefined);
-                if (object.ReferenceEquals(exceptionToThrow, objectDisposedException))
```

Comment:
I don't think CodeFormatter would revert the removal of object. I think we should do a round of running CodeFormatter on our code base as a separate task. It's been a while.